### ISSUE TITLE
Will `paddle_infer` make accuracy drop?

### ISSUE THREAD

**Issue by DrRyanHuang** [PaddlePaddle/Paddle#44600]
### 请提出你的问题 Please ask your question

Hi, I export an segmetation static model  with `PaddleSeg`, and used the static model with `paddle_infer`.
However, the accuracy(mIoU) has droped.

Here is our testing result:

The first two lines are the Paddle dynamic model directly running test, and the next three lines are the paddle_infer static model running test.
![d](https://user-images.githubusercontent.com/44900829/179449162-bdec4761-652a-45c1-8346-9894dd2a3d7e.png)

For consistency, all steps I used are from `PaddleSeg`, including reading data, calculating the mIoU and other postprocess. 
In other words, except inference, other operations are the same.

I also raise an issue in `PaddleSeg`, but get no help.
https://github.com/PaddlePaddle/PaddleSeg/issues/2325

So, Will `paddle_infer` make accuracy drop?

**Comment by thisjiang** [comment]
Hi~The `paddle_infer` will be no loss of accuracy without low accuracy, your problem may because of some features of  PaddleSeg. I have reminded the colleague of PaddleSeg to help you, please wait. Thanks.

**Comment by DrRyanHuang** [comment]
@thisjiang 
Hi, thank you for your help!!!!!!
This issue has been resolved, which is `mIoU` calculated differently.

------------
`mIoU` is not the mean of each image, but $$\frac{total\ area\ intersect}{total\ area\ union}$$.
